toolkit/tools/imagegen/attendedinstaller/views/hostnameview/hostnameview.go

```diff
@@ -4,14 +4,14 @@
 package hostnameview
 
 import (
-	"crypto/rand"
 	"fmt"
 	"strings"
 
 	"microsoft.com/pkggen/imagegen/attendedinstaller/primitives/navigationbar"
 	"microsoft.com/pkggen/imagegen/attendedinstaller/uitext"
 	"microsoft.com/pkggen/imagegen/attendedinstaller/uiutils"
 	"microsoft.com/pkggen/imagegen/configuration"
+	"microsoft.com/pkggen/internal/miscutils"
```

Comment:
>miscutils [](start = 32, length = 9)

I don't like the name `miscutils`. It's too generic and will probably end up accumulating all kinds of functions with no relation to each other. I'd suggest something with "random", "randomization", etc. in the name to make the purpose for these utils clearer.

Reply:
I figured that would be exactly what its used for, random stuff that doesn't merit its own package. No strong opinions either way though.

Reply:
Let's change it in that case. Often (at least from my experience) once people find something like this, they don't really think if they should create a separate package for their set of very theme-specific utils and just dump them to this one bag of whatever instead.

---
In reply to: [549878770](https://github.com/microsoft/CBL-Mariner/pull/479#discussion_r549878770) [](ancestors = 549878770)

Reply:
Fixed